### ISSUE TITLE
Multiple Electron windows and RxDB

### ISSUE THREAD

**Issue by ardoramor** [pubkey/rxdb#572]
## Case
I believe this to be a bug as RxDB officially supports Electron. However, documentation does not mention RxDB in multiple windows, which I assume works.

## Issue
I have an electron application with a `main` process, primary `renderer` process, and worker `renderer` process. When a user performs a certain action, primary process sends a request over IPC to main, which in turn forwards it onto the worker process.

The worker process performs an asynchronous operation and writes some data to a collection. When main process informs the primary process of the completed job, the primary process attempts to fetch the updated data and display it to the user. However, after the first initial fetch, all consecutive writes to the collection from the worker process are ignored and only the first entry is returned.

My guess is that RxDB stores some information about the last state of the collection. Because the update happens not in the same process, I assume that the instance in the primary process is not aware. However, I think this works in the browser, where different instances of the same page might be running and updates would be successfully propagating.

## Info
  - Environment: Electron
  - Adapter: IndexedDB
  - Stack: Webpack, Babel, VueJS

### Worker Process

```js
import * as Database from 'db'
  
export default {
   async populateBeers () {
         const db = await Database.get()

         db.beers.insert({
            id: 'beer_' + rand(),
            type: 'Hefe-Weissbier',
            name: 'Julius Echter'
         })

         ...
   }
}
```

### Primary Process

```js

import * as Database from 'db'
  
export default {
   ...,
   methods: {
      async loadBeers () {
         // Database.get() method instantiates DB once and then returns the same instance
         const db = await Database.get()

         db.beers
            .find()
            .exec()
            .then(console.dir)
      }
   }
}
```

**Comment by pubkey** [comment]
The electron-example also has multiple windows so this should work.
Which adapter do you use? Can you paste the code on how you create the database?

**Comment by ardoramor** [comment]
I've used the example to try and recreate my issue. Turns out `'multiInstance': false` was the culprit. From the [documentation](https://pubkey.github.io/rxdb/rx-database.html#multiinstance), I got the impression that `multiInstance` should be set to false in my application as I am using Electron with app written in VueJS. Each process has just one instance of the database running. I guess the following comment clarifies my mistake: 
> "...or a single-window electron-app"
